src/utils/Statistics.java

```diff
@@ -51,17 +54,18 @@ public void updateStats(ArrayVisualizer ArrayVisualizer) {
         this.sortHeading = ArrayVisualizer.getHeading();
         this.sortExtraHeading = ArrayVisualizer.getExtraHeading();
         int showUnique = Math.min(ArrayVisualizer.getUniqueItems(), ArrayVisualizer.getCurrentLength());
-        this.arrayLength = this.formatter.format(ArrayVisualizer.getCurrentLength()) + " Numbers" 
+        this.arrayLength = this.formatter.format(ArrayVisualizer.getCurrentLength()) + " Numbers"
         + ", " + this.formatter.format(showUnique) + " Unique";
-        
+
+        this.framerate = (1000.0 / frameTimeMillis) + " FPS";
```

Comment:
perhaps round the number, else you get the long repeating decimals problem

Reply:
I don't personally care about the long repeating decimals problem. And also the double round formatter is really big and slow.

Reply:
all you need to do is truncate the number, and besides looking really weird it gives a false sense of precision

Reply:
True, and fair